pkg/stub/handler.go

```diff
@@ -382,18 +411,24 @@ func (h *Handler) processFiles(dir string, files []os.FileInfo, opcfg *v1alpha1.
 		logrus.Printf("processing file %s from dir %s", file.Name(), dir)
 
 		if strings.HasSuffix(dir, "imagestreams") {
+			// allow setting of product dir as found in tmp/build/assests/operator as a way to skip as set of imagestreams
+			// i.e. .../php/imagestreams/...
+			// setting "php" will skip all the imagestreams under php/imagestreams
```

Comment:
I worry about this having unintended consequences.  The users setting skippedImageStreams don't have any insight into the underlying directory structure the operator is processing, so they could inadvertently skip things they don't intend to/don't understand.  The names are really the only thing they are exposed to.

Reply:
I admittedly was anticipating need here based on how ansible surfaced the on/off switches for which samples were installed.  It had higher level categories.

Sure, I can back this off, and we can wait and see if the request for skipping a group of content emerges, and based on how it manifests, warrants consideration.